tests/installation/concurrent-download.spec.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test } from './npmTest';
+import fs from 'fs';
+test.use({ isolateBrowsers: true });
+
+test('concurrent browser downloads should not clobber each other', async ({ exec }, testInfo) => {
+  await exec('npm init -y');
+  await exec('npm install playwright');
+  const numProcesses = 3;
+  await Promise.all(Array.from({ length: numProcesses }, (_, index) =>
+    exec('npx playwright install chromium', {
+      env: {
+        PLAYWRIGHT_BROWSERS_PATH: testInfo.outputPath(`browsers-${index}`),
+      }
+    })
+  ));
+  // Aggregate installed software across individual browser paths and verify expected components exist
```

Comment:
Why aggregate if you can simply check it in each directory? Just move the `installed` set and all 3 checks inside the loop - each installation should have all 3 binaries present.